package-lock.json

```diff
@@ -2113,14 +2064,14 @@
       }
     },
     "node_modules/@openzeppelin/contracts": {
-      "version": "4.8.1",
-      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts/-/contracts-4.8.1.tgz",
-      "integrity": "sha512-xQ6eUZl+RDyb/FiZe1h+U7qr/f4p/SrTSQcTPH2bjur3C5DbuW/zFgCU/b1P/xcIaEqJep+9ju4xDRi3rmChdQ=="
+      "version": "4.8.2",
+      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts/-/contracts-4.8.2.tgz",
+      "integrity": "sha512-kEUOgPQszC0fSYWpbh2kT94ltOJwj1qfT2DWo+zVttmGmf97JZ99LspePNaeeaLhCImaHVeBbjaQFZQn7+Zc5g=="
     },
     "node_modules/@openzeppelin/contracts-upgradeable": {
-      "version": "4.7.1",
-      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts-upgradeable/-/contracts-upgradeable-4.7.1.tgz",
-      "integrity": "sha512-5EFiZld3DYFd8aTL8eeMnhnaWh1/oXLXFNuFMrgF3b1DNPshF3LCyO7VR6lc+gac2URJ0BlVcZoCfkk/3MoEfg=="
+      "version": "4.8.2",
```

Comment:
@openzeppelin/contracts-upgradeable should remain @ 4.7.1 
They made some breaking changes.

Reply:
fixed